WordPress/Resources/HTML/richCommentStyle.css

```diff
@@ -0,0 +1,189 @@
+:root {
+    /* informs WebKit that this template supports both appearance modes. */
+    color-scheme: light dark;
+
+    /* custom variables. */
+    --wordpress-blue-color: rgba(6, 117, 196, 1);
+    --mention-background-color: rgba(2, 103, 255, .1);
+    --monospace-font: ui-monospace, monospace;
+}
+
+/* overrides for dark color scheme. */
+@media(prefers-color-scheme: dark) {
+    :root {
+        --wordpress-blue-color: rgba(57, 156, 227, 1);
+        --mention-background-color: rgba(2, 103, 255, .15);
+    }
+}
+
+/* disable WebKit text inflation algorithm to prevent text size increase on orientation change. */
+html {
+    -webkit-text-size-adjust: none;
+}
+
+/* use Apple's standard style to match the native look. */
+body {
+    font: -apple-system-body;
+    color: -apple-system-label;
+    background-color: -apple-system-background;
+}
+
+/* get rid of the container default margins and paddings. */
+html, body {
+    margin: 0;
+    padding: 0;
+}
+
+/* remove the bottom margin of ANY element that's placed last within the <body> tag. */
+body :last-child {
+    margin-bottom: 0;
+}
+
+p {
+    word-wrap: break-word;
+}
+
+/* remove the default left/right margin for figures, as some elements (tables, images) are often embedded in <figure>. */
+figure {
+    margin: 0
+}
+
+/** LINKS **/
+
+a {
+    color: var(--wordpress-blue-color);
+}
+
+a.mention {
+    text-decoration: none;
+    background-color: var(--mention-background-color);
+    padding: 0 3px;
+    -webkit-border-radius: 3pt;
+}
+
+/** IMAGES **/
+
+.wp-block-image img {
```

Comment:
We might need to have a rule for regular `img` tags that aren't part of gutenberg blocks. I tested with a non-gutenberg comment that had an image tag. The portrait image had some issues with layout. The width was constrained to width of the web view but the aspect ratio was lost resulting in the image having it's natural height. 
I tried adding 
```
img {
    margin: 0 auto; /* align center */
    width: auto;
    max-width: 100vw;
    height: auto;
}
```
This seemed to fix the issue with the image tho I'm not sure if it's all necessary or would have other side effects we don't want.

Reply:
Good catch again! 🙂 Will fix this in the upcoming commit.

Reply:
Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/17129/commits/1337847fbee3cd3240a33469a7a30c25b4d643f6.